webui/tests/bin/createUserAndExperiments.py

```diff
@@ -26,7 +27,17 @@
 # but the `setsid` command does not seem to be generally available on macOS.)
 # FIXME The script is blocking here on macOS. We disable this and its corresponding tests for the
 # time being until it's fixed.
-# p = pexpect.spawn("det", ["-u", "admin", "u", "create", "hoid"])
-# p.expect("Password.*:")
-# p.sendline("")
-# p.wait()
+p = pexpect.spawn("det", ["-u", "admin", "u", "create", "hoid"])
+p.expect("Password.*:")
```

Comment:
seems like the `det u create` behavior has changed. it no longer expects you to enter a password. at least for passwordless account.

you would instead pair it with `det -u admin user change-password abc` to change the password which asks for it twice.

Reply:
wondering how it passed the tests ..

Reply:
it'd be great to add an accompanying e2e tests that logs in with the newly created user and a present non-zero passowrd

Reply:
good call, updated to create a new user with a password.